main.yml

```diff
@@ -435,8 +435,32 @@ subscriptions:
   frontend:
     paths:
       - /staging/subscriptions
+    sub_apps:
+      - id: subscriptions-rhel
   git_repo: https://github.com/RedHatInsights/curiosity-frontend
 
+subscriptions-rhel:
+  title: Red Hat Enterprise Linux
+  frontend:
+    sub_apps:
+      - id: all
+        title: All
+        reload: rhel/all
```

Comment:
With reload you are loosing SPA experience and each nav click is refreshed, unless you want to change the URL I wouldn't add it here. Also `rhel` is quite unlucky because we already have rhel application. How exactly do you want your URL to look like? And how your nav is going to look like?

Reply:
#### Regarding the path/route question:
Indeed, we noticed the heavy refresh on navigation. The original application path we were aiming for would based on our route param, so everything would follow similar to

-`[domain]/beta/subscriptions/rhel/?variant`, variant being the "sub-apps" so in the case of `all`
- `[domain]/beta/subscriptions/rhel/all` or optional, and then continuing in that style for the remaining `sub-apps`

- Specifically we were having trouble getting the routing to work when we removed the `reload`, so if there's an alternative approach we're up for it
- Plus, if we have a recommendation for something other than `rhel` I don't think the team would be opposed to altering that path/route

Reply:
Catching up with the team regarding `rhel` alternatives

Reply:
Ok @kruai @karelhala going to switch over to using `rhel` spelled out `redhatenterpriselinux`, seems to be the most direct alteration. Updating the PR

Reply:
Since you already know the "bundle" your app is going to live, it might be good to introduce it as well. You are currently using `staging` which is for temporary apps untill they know where they are going to be included and such.

So your navigation can look like:
```
[Home] | Subscription Watch
--------------------------------------
* Red Hat Enterprise Linux
  * All
  * ARM
  * IBM Power
  * IBM Z systems
  * x86
```

If that is the case we have to introduce new bundle and use this in it. Creating new bundle is fairly easy and can be done here, you can check how this was done for `Settings` https://github.com/RedHatInsights/cloud-services-config/pull/94/files